### ISSUE TITLE
Whatsapp Button

### ISSUE THREAD

**Issue by Tobias-Lauerbach** [Chooloo/koler#447]
### Checklist

- [x] I am using the latest version - 1.4.6 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Not Only Whatsapp Support, or te option th hide the open in Whatspp Button if you don´t Whatsapp (The easy Way)
It would be great to open Contacts in Threema :-)
#### Is your feature request related to a problem? Please describe it

It shows a not working Whatsapp button if you don´t have Whatsapp ..

#### How will you/everyone benefit from this feature?

Every Privacy Orientated User will aprecheate no having a whatsapp button In a FOSS app (If they don´t use Whatsapp)

**Comment by Gottarocket** [comment]
Signal ? I believe it needs more permission to access all apps on device but if there isnt best to add button finding default messenger app or show button based on app we use. Or option to manually select in settings

**Comment by xdubx** [comment]
Yes it works with links. I don't have WhatsApp installed and it open my browser. 

I support the option to change the button or disable it.

**Comment by gerhardbeck** [comment]
As I just have threema I`m supporting the possibility to choose an app

**Comment by xdubx** [comment]
I'am creating a pr in the next time. Implementing at the moment :D

**Comment by xdubx** [comment]
iam created a pr https://github.com/Chooloo/koler/pull/494

**Comment by aluyama** [comment]
@xdubx please check this https://github.com/Chooloo/koler/issues/524
